client/js/uploader.api.js

```diff
@@ -326,7 +326,7 @@
 
             this._templating.updateProgress(id, loaded, total);
 
-            if (Math.round(loaded / total * 100) === 100) {
+            if (total === 0 || Math.round(loaded / total * 100) === 100) {
```

Comment:
Why was this added? It's not clear to me what consequences are associated with this change.